core/tests/protractor/accessibility.js

```diff
@@ -54,3 +54,40 @@ describe('Cache Slugs', function() {
     general.checkConsoleErrorsExist(expectedErrors);
   });
 });
+
+describe('Meta Tags', function() {
```

Comment:
This is not a web accessibility test. Perhaps put it in navigation.js or create a new file (if the latter, make sure to include it in protractor.conf.js and .travis.yml).

Reply:
Done